### ISSUE TITLE
Changing managedPolicies on aws-sso.yml not triggering update

### ISSUE THREAD

**Issue by donatoaz** [org-formation/org-formation-cli#217]
### Subject of the issue

Changing the managedPolicies peroperty on sso task is not triggering an update.

```diff
diff --git a/tasks/sso/_tasks.yml b/tasks/sso/_tasks.yml
index c122caf..68a3822 100644
--- a/tasks/sso/_tasks.yml
+++ b/tasks/sso/_tasks.yml
@@ -57,7 +57,7 @@ SsoDeveloper:
     instanceArn: !Ref instanceArn
     principalId: !Ref developerGroup
     permissionSetName: 'Developer'
-    managedPolicies: [ 'arn:aws:iam::aws:policy/PowerUserAccess' ]
+    managedPolicies: [ 'arn:aws:iam::aws:policy/AdministratorAccess' ]
     sessionDuration: 'PT12H'
```

The full repo here: https://github.com/donatoaz/org-formation

### Your environment
* version of org-foramtion (ofn --version) 0.9.14
* version of node (node --version) v10.15.1
* which OS/distro osx

### Steps to reproduce
Tell us how to reproduce this issue. Please provide ofn projct files if possible,
you can use [this template](https://plnkr.co/edit/m568SDw2KPufQsUl?p=preview) as a base.
https://plnkr.co/edit/m568SDw2KPufQsUl

I deployed a permission set using the reference aws-sso.yml file and tried to change the developer permissions from PowerUser to Administrator.

### Expected behaviour

The permission set to be updated with the new managed policy

### Actual behaviour

```
org-formation perform-tasks ./_tasks.yml --organization-file ./organization.yml --max-concurrent-stacks 50 --max-concurrent-tasks 1 --print-stack --verbose --profile janitor
DEBG: Task OrganizationUpdate execute starting...
INFO: Executing: update-organization organization.yml.
INFO: organization up to date, no work to be done.
INFO: Task OrganizationUpdate execute successful.
...
DEBG: Task AwsSso execute starting...
INFO: Executing: include tasks/sso/_tasks.yml.
DEBG: Task SsoAdministrator execute starting...
INFO: Executing: update-stacks tasks/sso/aws-sso.yml donatoaz-sso-admin.
DEBG: Setting build action on stack donatoaz-sso-admin for 846282225459/us-east-1 to None - hash matches stored target. (846282225459 = ManagementAccount)
INFO: Stack donatoaz-sso-admin already up to date.
INFO: Task SsoAdministrator execute successful.
DEBG: Task SsoDeveloper execute starting...
INFO: Executing: update-stacks tasks/sso/aws-sso.yml donatoaz-sso-developer.
DEBG: Setting build action on stack donatoaz-sso-developer for 846282225459/us-east-1 to None - hash matches stored target. (846282225459 = ManagementAccount)
INFO: Stack donatoaz-sso-developer already up to date.
INFO: Task SsoDeveloper execute successful.
DEBG: Task SsoReadOnly execute starting...
INFO: Executing: update-stacks tasks/sso/aws-sso.yml donatoaz-sso-viewer.
DEBG: Setting build action on stack donatoaz-sso-viewer for 846282225459/us-east-1 to None - hash matches stored target. (846282225459 = ManagementAccount)
INFO: Stack donatoaz-sso-viewer already up to date.
INFO: Task SsoReadOnly execute successful.
DEBG: Done performing task(s).
INFO: Task AwsSso execute successful.
DEBG: Done performing task(s).
DEBG: putting object to S3: 
{
  "Bucket": "organization-formation-846282225459",
  "Key": "state.json"
} (846282225459 = ManagementAccount)
```

**Comment by OlafConijn** [comment]
Thanks for logging this! This is a known issue and solved in a more recent beta. 0.9.15-beta.11. This beta is stable and looking forward to release soon.

Hope this helps.

Would have pasted more info but currently on my phone and wanted to help you get unstuck.

**Comment by OlafConijn** [comment]
thanks! fixed in release v0.9.15
